packages/terra-clinical-onset-picker/src/terra-dev-site/test/clinical-onset-picker/FiveYears.test.jsx

```diff
@@ -10,8 +10,8 @@ class FiveYears extends React.Component {
     moment.now = () => new Date(2016, 8, 20);
 
     this.state = {
-      precision: OnsetPicker.Opts.Precisions.BEFORE,
-      granularity: OnsetPicker.Opts.Granularities.YEAR,
+      precision: 'before',
+      granularity: 'year',
```

Comment:
I thought these were removed?

Reply:
Again, it was just the default values that were removed. Can still set the props.